packages/uicore/src/components/Tabs/Tabs.tsx

```diff
@@ -39,18 +39,22 @@ function tab(props: TabProps) {
 }
 
 function Tabs(props: TabsProps): React.ReactElement {
-  const { renderAllTabPanels, vertical, children, tabList = [], ...rest } = props
+  const { renderAllTabPanels, vertical, children, className, tabList = [], ...rest } = props
   const hasIcons = tabList.findIndex(tabProp => tabProp.iconProps?.name) !== -1
   return (
     <BpTabs
       {...rest}
```

Comment:
`className` was being extracted out in `rest` prop and further being overridden on line 49. This is now fixed.